tests/custom_managers/models.py

```diff
@@ -155,6 +155,22 @@ class Meta:
         base_manager_name = 'annotated_objects'
 
 
+class ConfusedBook(models.Model):
+    title = models.CharField(max_length=50)
+    author = models.CharField(max_length=30)
+    favorite_things = GenericRelation(
+        Person,
+        content_type_field='favorite_thing_type',
+        object_id_field='favorite_thing_id',
+    )
+    favourite_things = GenericRelation(
+        FunPerson,
+        content_type_field='favorite_thing_type',
+        object_id_field='favorite_thing_id',
+        related_query_name="favorite_things",
```

Comment:
It was really hard for me to see how `favorite_things` and `favourite_things` collide, so I renamed them.

Reply:
Works for me; by that point I was just trying to figure out the scenario in which the convoluted collision would occur, and test it :)